docs/guide/distancesensor-sensors.md

```diff
@@ -0,0 +1,146 @@
+## DistanceSensor Sensors
+
+Some predefined [DistanceSensor](../reference/distancesensor.md) models are present in Webots such as some infrared sensors.
+
+Nevertheless, adding a generic [DistanceSensor](../reference/distancesensor.md) node is sufficient for most of the applications because they are customizable (resolution, lookupTable, aperture, etc.).
+
+### Sharp
+
+#### Sharp GP2D120
+
+%figure "Sharp GP2D120 model"
+
+![sharp_GP2D120.png](images/sensors/sharp_GP2D120.png)
+
+%end
+
+The [Sharp GP2D120](https://www.pololu.com/file/0J157/GP2D120-DATA-SHEET.pdf) is a distance measuring sensor with integrated signal processing and analog voltage output.
+This device outputs the voltage corresponding to the detection distance.
+
+The model has the following specifications:
+
+- `effective range`: 4 to 30 [cm]
+- `output type`: analog
+- `response time`: 39 [ms]
+- `start up delay`: 44 [ms]
+- `average consumption`: 33 [mA]
+
+```
+PROTO SharpGP2D120 {
+  SFVec3f     translation  0 0 0
+  SFRotation  rotation     0 1 0 0
+  SFString    name         "Sharp's IR sensor GP2D120"
+  SFString    model        "GP2D120"
+}
+```
+
+The `lookupTable` field of the [DistanceSensor](../reference/distancesensor.md) is already implemented according to it characteristics found in his [datasheet](https://www.pololu.com/file/0J157/GP2D120-DATA-SHEET.pdf).
+
+The function `wb_distance_sensor_get_value` return the voltage/intensity of the measurement. To convert these values, use the following formulas:
+- Given meter, the result expected is in voltage: `y(x) = 0.5131*x^(-0.5735)-0.6143`
```

Comment:
this is the formula to convert from meters to voltage, is it correct?
Why do not simply write something like this:
```suggestion
- Convert meters to voltage: `y(x) = 0.5131*x^(-0.5735)-0.6143`
```

Same for the next line.